velox/type/Type.h

```diff
@@ -2003,11 +2004,27 @@ class CastOperator;
 using CastOperatorPtr = std::shared_ptr<const CastOperator>;
 } // namespace exec
 
+/// Forward declaration.
+class variant;
+class AbstractInputGenerator;
+
+using AbstractInputGeneratorPtr = std::shared_ptr<AbstractInputGenerator>;
+using FuzzerGenerator = folly::detail::DefaultGenerator;
+
+struct InputGeneratorConfig {
+  size_t seed_;
+  double nullRatio_;
+
+  // Type of data represented by JSON. This config should be ignored by non-JSON
+  // input generators.
+  const TypePtr& representedType_;
```

Comment:
Could we just let the JSON Generator call velox::randType(...) with a FuzzerGenerator created from seed_?

Reply:
Alternatively, JSON really only supports a handful of types: string, integer, float, object (map), array, and boolean (any of which can be null), so JSON could just randomly construct one of those if you don't want to link against VectorFuzzer in the Presto Type code.